config/navigation.yaml

```diff
@@ -36,7 +37,7 @@ waypoint:
 recovery:
   stop_thresh: 0.2
   drive_fwd_thresh: 0.34
-  recovery_distance: 1.0 # distance the rover moves back in each leg of j-turn
+  recovery_distance: 2.0 # distance the rover moves back in each leg of j-turn
```

Comment:
Should this be in the failure recovery PR?

Reply:
technically yea but that was merged some time ago (the one thats in is just detection) and if you don't mind I think its easier to just leave it